element-hq/element-x-android
# Add setting to compress image and video

## Content


Add a new setting in the advanced section to compress image and video. Setting is disabled by default. When enabled, the image and the video will be compressed.

## Motivation and context


Closes #3651

## Screenshots / GIFs


See recorded ones.

## Tests



- Send an image
- Observe the the image is sent with the original size
- Enable the new setting
- Send the image again
- Observe that the image quality is lower, but the image size is also lower.

## Tested devices

- OS version(s):

## Checklist
